src/Pos/Generator/Block/Mode.hs

```diff
@@ -0,0 +1,245 @@
+-- | Execution mode used by blockchain generator.
+
+module Pos.Generator.Block.Mode
+       ( MonadBlockGenBase
+       , MonadBlockGen
+       , BlockGenContext (..)
+       , BlockGenMode
+       , mkBlockGenContext
+
+       , InitBlockGenContext (..) -- useless
```

Comment:
Yes, it is. I think i did it because otherwise record fields were not used. Will prefixing them with `_` work?